### PR TITLE
Use StartEnigma name instead of Startup

### PR DESCRIPTION
**Pull request by persianpros**
- This is a better name as we're starting enigma not something else, Startup is very general and common as we may ask startup what?

- Add StartEnigma in debug lines so we understand StartEnigma is producing them.

- Correct all .po and other related .py files which advertise this file.

### CODE REVIEW

**Comment by RobvanderDoes** [comment]
I presume you also relabelled the 'Start' button in your car to 'Start the engine (in this car)'?
Otherwise you might be wondering what the button would start 🤣

**Comment by persianpros** [comment]
@RobvanderDoes Do you read your car's OBD-II logs? Don't be stupid please.

**Comment by RobvanderDoes** [comment]
Yes, I do look into the OBD details of my car. Even without being stupid.

**Comment by IanSav** [comment]
Yes, I would like the Start button to be properly labelled in case it is next to the Start button for any of the other car components like the radio or the air conditioner.  If there is no other reasonable option in the context then by all means simplify the label.

In the context of a computer there are many things that may be started.  Some of those tasks may be built in Python so having the Python code named appropriately is a very reasonable proposal.

The comment seems to be quite uncharacteristic for the poster.

**Comment by persianpros** [comment]
@RobvanderDoes 

Please google this: sonata engine start button (example of one car in my country)

Even in cars they do have it, not sure why you compared my commit with cars and laughed at me but please solve your personal problems then come and say something :+1:

**Comment by RobvanderDoes** [comment]
🤣

**Comment by Dima73** [comment]
StartEnigma probably the correct name will be.
I agree with it.

**Comment by ccs-ccs** [comment]
.... so not StartEnigma2 ??

**Comment by IanSav** [comment]
If you use the "StartEnigma2" name then all this will need to be changed again for the next generation of Enigma.  ;)  The name "StartEnigma" tells us what it is doing without needing to specify a version number.

I think this is a worthy change.

**Comment by SimonCapewell** [comment]
I'm finding it remarkable how much debate my name change commit has generated given that for 17 years, nobody cared that the startup module was called mytest or did anything about it.

**Comment by Ev0-BH** [comment]
Begs the question, why bother changing it at all ?

File: lib/python/Screens/Standby.py [hunk 1]
```diff
@@ -247,7 +247,7 @@ def __init__(self, session, retvalue=QUIT_SHUTDOWN):
 			QUIT_REBOOT: _("Your receiver is rebooting"),
 			QUIT_RESTART: _("The user interface of your receiver is restarting"),
 			QUIT_UPGRADE_FP: _("Your frontprocessor will be updated\nPlease wait until your receiver reboots\nThis may take a few minutes"),
-			QUIT_ERROR_RESTART: _("The user interface of your receiver is restarting\ndue to an error in mytest.py"),
+			QUIT_ERROR_RESTART: _("The user interface of your receiver is restarting\ndue to an error in StartEnigma.py"),
```
Comment (SimonCapewell) [inline lib/python/Screens/Standby.py:250]: I can't see anything using this. There are no calls to `TryQuitMainloop`  that pass in either `QUIT_ERROR_RESTART` or `5`. Consider removing it.

**Comment by SimonCapewell** [comment]
It was part of a commit that moved the py files in the root to their correct location, so you can point e2 at at the source tree and actually have it launch. `mytest` was clearly a temporary name copied from some kind of example in 2004 so it made sense to rename it rather than perpetuate the misnomer. Now it seems there's some confusion over what startup.py could possibily be starting.

**Comment by persianpros** [comment]
@SimonCapewell just to let you know this isn't your idea, I did it sooner: https://github.com/OpenVisionE2/enigma2-openvision/commit/a01cb68e4e703cd94d8c8986a09f4662cd49832d and was thinking about it years before so your statement is wrong.

Anyway this isn't a contest and I'm just offering what I think is better :+1:

**Comment by IanSav** [comment]
> I'm finding it remarkable how much debate my name change commit has generated given that for 17 years, nobody cared that the startup module was called mytest or did anything about it.

I actually proposed this many months ago to OpenViX, Tony coded it but we couldn't get it accepted.  OpenVision adopted the change as soon as I suggested it there.

EDIT: I see now that this is something about which Persian was already thinking.  Perhaps my suggestion pushed him over the edge to implement it.

**Comment by IanSav** [comment]
> Begs the question, why bother changing it at all ?

The change is appropriate because it better documents what it does.

**Comment by SimonCapewell** [comment]
@persianpros Sigh. More than one person can independently come up with the same idea you know. Especially when it's so bleedingly obvious you can barely call it an idea. If you really want to start talking dates, I started a conversation with an example branch to move the root python files on the OpenVix forums on 23 May last year.

**Comment by IanSav** [comment]
There is enough angst around here, why not get this merged and move on.

**Comment by SimonCapewell** [comment]
Incidentally, for core changes like renaming mytest, OpenVix prefer to have OpenPLi accept the change first so that merges are less painful. That's probably why you couldn't get it accepted. It's indirectly the reason why it took me nearly a year to get round to submitting a PR to OpenPLi - building and installing a second distro wasn't something I wanted to delve into until recently.

**Review by AbuBaniaz — APPROVED** [review]

**Comment by IanSav** [comment]
@SimonCapewell, I understand the OpenViX preference.  Unfortunately I found it inconsistently applied.  Recently I feel that OpenViX has broken much of the interoperability toward which a few of us were working.

I really don't understand why all the different images exist, or continue to justify their existence, when there are so many dependencies on the efforts and state of progress in other images.

**Comment by SimonCapewell** [comment]
Inertia. As I found recently with a fix for recordings hanging a variety of boxes, what should've been a simple patch to give an immediate fix to the problem disintegrated into months of tedious discussion over while loop exit strategies and endless test builds. That kind of thing makes contributing to a larger single distro rather unattractive.

What things have OpenVix broken/diverged on?

**Comment by IanSav** [comment]
One of the main issues where I believe OpenViX has lost its way is in all the changes being made to support what I believe is a failed design for skin scaling.  There are a number of other changes that made code on different images different where I had them almost identical.  There is also the issue of OpenViX publicly ridiculing me and my work yet rather than revert the code they continue to use and extend its use.  The public posts don't really correlate with their actions.  That said, I don't really want to look backwards.  I prefer to move forwards.

I am still working on all the projects I proposed to OpenViX but I am now delivering them on OpenVision.  In general I am simplifying code yet making it more flexible and powerful.  I am also making more code shared within the image to make maintenance and support significantly easier.  The new Information.py module I proposed for OpenViX and was forbidden from implementing is now delivered in OpenVision.  It has proven to be successful, popular and well received by users.  I have also delivered the new help system that OpenViX failed to progress.  I am now working on a significantly simplified and enhanced remote control system.  Next I hope to deliver the skin scaling design that I tried to deliver on OpenViX.  The new Storage, Network and International updates are still being designed and progressed but are a little further down the track.

**Comment by SimonCapewell** [comment]
Does OpenPLi figure in these projects?

**Comment by IanSav** [comment]
Very much so.  OpenVision is closely related to OpenPLi.  I am developing and workshopping the changes on OpenVision as the development environment is very supportive.  This environment gives me an opportunity to take some risks with new ideas.  It allows the ideas to be coded, tried out and reviewed.  Sometimes changes are requested or improvements are suggested.  My hope is that OpenPLi will also accept the changes once they stabilise.

I note that OpenViX is now also taking my code developments from OpenVision.  An OpenATV member has joined our development discussion channel and is following our work and may also directly take changes from the OpenVision developments.

Contrary to some comments being made about OpenVision, we are open and very prepared to cooperate with other teams that want to cooperate with us.

**Comment by Hains** [comment]
@IanSav  The build-environment is closed! Yeah there is a public oe-core, but that's frozen, obsolete and no support.
So Open Vison is Closed, as you can't build an image, unless you're a developer or you're added to the  Certified Compilers list.. OMG...
And even then you have to deal with the restrictions of the so called Persian Prince license.
Enigma2 is open again (I'm sure) against his will.

So I think it's better not to cooperate with other teams.

Thank you Persian Prince for ruining freedom and open source.

**Comment by persianpros** [comment]
Again another place to say something to me, I should not come near this places, thank you all.

**Comment by IanSav** [comment]
I think this pull request should be restored.  It is worthwhile.  We don't know why it was taking so long to merge.  Unless it is formally rejected I think we should wait for comment from the OpenPLi team.

**Comment by IanSav** [comment]
@Hains, I only work mainly in the Enigma2 repository and it is fully open and all my code is open and available to all.

I may be missing your point or reason you need/desire to build OpenVision but your post verges on a hostile attack on Persian.  This serves no productive purpose.

I should also mention that OpenVision is working on getting a new build server and we hope to release officially built images for everyone to use.  (Remember that we are a totally unfunded team so getting some things done will take longer.)

**Comment by Hains** [comment]
Yeah i know Enigma2 is fully open.
The Open Vision core is closed, only a public fake ov-oe-core repo.
The fact that you don't see further than Enigma2 doesn't making it an open image.

I don't need the desire to build an Open Vision image, that's not the point.

Attack? What attack? It's a fact .
But I'll stop now, this is indeed not the place and I'm done with it actually.

**Pull request closed without merging**